### PR TITLE
add getInstrumentsPath(), update to teen_process

### PR DESCRIPTION
**Pull request by Jonahss**

### CODE REVIEW

File: test/xcode-specs.js [hunk 1]
```diff
@@ -89,4 +89,12 @@ describe('xcode @skip-linux', () => {
     (typeof devices).should.equal('object');
   });
 
+  it('should get the path to instruments binary', async() => {
+    let instrumentsPath = await xcode.getInstrumentsPath();
+
+    should.exist(instrumentsPath);
+    (typeof instrumentsPath).should.equal('string');
```
Comment (jlipps) [inline test/xcode-specs.js:96]: i guess the way we do this now is `instrumentsPath.should.be.a('string');`

**Comment by jlipps** [comment]
looks great! one tiny change then feel free to merge/publish

**Comment by sebv** [comment]
I've setup Travis, so we see which test are unit/e2e. No worry if there are only e2e tests for now, I'll setup some coverage and we can write the unit tests later.

**Comment by Jonahss** [comment]
closing this pr in favor of #15

**Pull request closed without merging**